### ISSUE TITLE
Expand IMT list

### ISSUE THREAD

**Issue by emthompson-usgs** [cbworden/mmi_ddistributions#1]
Current data only includes MMI, PGA, PGV, and 3 SAs. Eventually we should expand this to include at least include a broader range of spectrals. Maybe include other IMTs like AI, duration, etc.

**Comment by cbworden** [comment]
Yeah, that would be good. The data in the current file was pulled from the ShakeMap amps that the networks produce. If we want to expand the SAs, we'll have to process waveforms. Fortunately we have the code to do that. But there'd be some effort in pulling together all the waveforms.
